tests/baselines/reference/api/typescript.d.ts

```diff
@@ -2266,7 +2266,7 @@ declare namespace ts {
         AlwaysStrict = 64,
         PriorityImpliesCombination = 28
     }
-    interface JsFileExtensionInfo {
+    interface FileExtensionInfo {
```

Comment:
this is a breaking change really. consider not changing the name, or adding a type alias `type JsFileExtensionInfo  = FileExtensionInfo`

Reply:
Added type.